packages/calcite-components/src/components/chip/chip.scss

```diff
@@ -78,7 +91,7 @@
 }
 
 :host([scale="s"]) .container.image--slotted {
-  padding-inline-start: calc(theme("spacing[0.5]") / 2);
+  padding-inline-start: calc(var(--calcite-internal-chip-spacing-s) / 2);
```

Comment:
instead of adding style props to each rule based on :host props, can you reassign a single CSS variable instead?

Reply:
Updated all the style-specific ones based on appearance / kind. The spacing ones will need to be revisited as they currently are not always based on reflected attributes, but also internal conditional classes. Should be addressed with https://github.com/Esri/calcite-design-system/issues/7425.